pkgs/tools/admin/google-cloud-sdk/default.nix

```diff
@@ -21,7 +21,20 @@
 }:
 
 let
-  pythonEnv = python.withPackages (
+  python' = python.override {
+    self = python';
+    packageOverrides = final: prev: {
+      pyopenssl = prev.pyopenssl.overridePythonAttrs (o: {
+        patches = [
+          # fix for oauth2client used by google-cloud-sdk
+          # see: https://github.com/googleapis/google-api-python-client/issues/2554
+          ./revert-1374-Remove-APIs-that-have-been-deprecated-for-a-year.patch
```

Comment:
Can we use `fetchpatch` with `revert = true`?
Pinning to an old version of pyopenssl is also an option but probably not a good idea security-wise.

Reply:
Applying it on v25.0.0 requiers conflict resolution, so it won't work.

Reply:
I've oppened the separate [PR](https://github.com/NixOS/nixpkgs/pull/384452) for google-cloud-sdk - targeting master.